jetty-util/src/main/java/org/eclipse/jetty/util/DateCache.java

```diff
@@ -245,17 +242,14 @@ protected Tick formatTick(long now)
     {
         long seconds = now / 1000;
 
-        // Synchronize to protect _tzFormat
-        synchronized (this)
+        // recheck the tick, to save multiple formats
+        if (_tick==null || _tick._seconds!=seconds)
         {
-            // recheck the tick, to save multiple formats
-            if (_tick==null || _tick._seconds!=seconds)
-            {
-                String s= _tzFormat.format(new Date(now));
-                return _tick=new Tick(seconds,s);
-            }
-            return _tick;
+            String s = ZonedDateTime.ofInstant(Instant.now(),_zoneId).format(_tzFormat);
+            return _tick=new Tick(seconds,s);
```

Comment:
without the synchronise, you cannot use _tick directly like this in multiple references as the _tick may change in another thread between references, so you have created a race here.  

You need to assign to a local variable  `Tick tick = _tick;` and then use `tick` for the logic.